python-bareos/bareos/bsock/lowlevel.py

```diff
@@ -46,17 +46,21 @@
 from bareos.util.password import Password
 import bareos.exceptions
 
-# Try to load the sslpsk module,
-# with implement TLS-PSK (Transport Layer Security - Pre-Shared-Key)
-# on top of the ssl module.
-# If it is not available, we continue anyway,
+# The ssl module support TLS-PSK (Transport Layer Security - Pre-Shared-Key)
+# since Python >= 3.13.
+# For some older Python versions, the TLS-PSK functionality
+# can be added by the sslpsk module,
+# with implement TLS-PSK on top of the ssl module.
+# If it is also not available, we continue anyway,
 # but don't use TLS-PSK.
-try:
-    import sslpsk
-except ImportError:
-    warnings.warn(
-        "Connection encryption via TLS-PSK is not available, as the module sslpsk is not installed."
-    )
+if not getattr(ssl, "HAS_PSK", False):
+    try:
+        import sslpsk
+    except ImportError:
+        warnings.warn(
+            "Connection encryption via TLS-PSK is not available "
+            "(TLS-PSK is not available in the ssl module and the extra module sslpsk is not installed)."
```

Comment:
I found the warning too long on terminal output.

Reply:
I've shorted the warning.